components/utils/src/fs.rs

```diff
@@ -48,9 +48,7 @@ pub fn create_directory(path: &Path) -> Result<()> {
 /// Return the content of a file, with error handling added
 pub fn read_file(path: &Path) -> Result<String> {
     let mut content = String::new();
-    File::open(path)
-        .map_err(|e| Error::chain(format!("Failed to open '{:?}'", path.display()), e))?
```

Comment:
We don't want to remove that, otherwise we don't know what failed in the various calls across the project. It's ok if the file is mentioned twice but at least we're sure we get the path.

Reply:
Very good point. Maybe read_file() can also take a name argument so it can produce all the errors itself?

- Failed to load config from /foo/config.toml
- Failed to load page from /foo/content/bar.md
- Failed to load theme foobar from /foo/themes/foobar/theme.toml
- etc..

Reply:
I'm not sure it's worth passing another argument

Reply:
Can you revert that? It's ok to merge otherwise I think

Reply:
OK i just removed the extra quotes printed due to debug output.